website/siteConfig.js

```diff
@@ -35,7 +35,7 @@ const users = loadYaml("./data/users.yml").map(user => ({
   image: `/img/users/${user.logo}`,
 }));
 
-const sponsorsManual = loadYaml("./data/sponsors.yml").map(sponsor => ({
+const sponsorsManual = (loadYaml("./data/sponsors.yml") || []).map(sponsor => ({
```

Comment:
Do we still need this `sponsorsManual` logic?

Reply:
Well maybe we might need it again in the future

Reply:
Yeah the only reason why I kept!